src/test/java/uk/gov/digital/ho/hocs/workflow/processes/DCU_MIN_MinisterSignOff.java

```diff
@@ -0,0 +1,115 @@
+import org.junit.ClassRule;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+import uk.gov.digital.ho.hocs.workflow.BpmnService;
+
+import static org.camunda.bpm.engine.test.assertions.ProcessEngineTests.withVariables;
+import static org.mockito.Mockito.*;
+
+@RunWith(MockitoJUnitRunner.class)
+@Deployment(resources = "processes/DCU_MIN_MINISTER_SIGN_OFF.bpmn")
+public class DCU_MIN_MinisterSignOff {
+
+    public static final String APPROVE_MINISTER_SIGN_OFF = "UserTask_0eagz4p";
+    public static final String DCU_MIN_MINISTER_SIGN_OFF_END = "DCU_MIN_MINISTER_SIGN_OFF_END";
+    public static final String VALIDATE_REJECTION_NOTE = "UserTask_04hin8c";
+    public static final String DCU_MIN_MINISTER_SIGN_OFF = "DCU_MIN_MINISTER_SIGN_OFF";
+    public static final String VALIDATE_NOT_APPLICABLE = "UserTask_1j77wfm";
+    @Rule
+    @ClassRule
+    public static TestCoverageProcessEngineRule rule = TestCoverageProcessEngineRuleBuilder.create()
+            .assertClassCoverageAtLeast(0.86)
+            .build();
+
+    @Rule
+    public ProcessEngineRule processEngineRule = new ProcessEngineRule();
+    @Mock
+    BpmnService bpmnService;
+    @Mock
+    private ProcessScenario dcuMinSignOffProcess;
+
+    @Before
+    public void defaultScenario() {
+        Mocks.register("bpmnService", bpmnService);
+    }
+
+    @Test
+    public void acceptSignOffScenario() {
+
+        when(dcuMinSignOffProcess.waitsAtUserTask(APPROVE_MINISTER_SIGN_OFF))
+                .thenReturn(task -> task.complete(withVariables("valid", true, "MinisterSignOffDecision", "ACCEPT")));
+
+        Scenario.run(dcuMinSignOffProcess)
+                .startByKey(DCU_MIN_MINISTER_SIGN_OFF)
+                .execute();
+
+        verify(dcuMinSignOffProcess)
+                .hasFinished(DCU_MIN_MINISTER_SIGN_OFF_END);
+
+        verify(dcuMinSignOffProcess, never()).hasFinished(VALIDATE_REJECTION_NOTE);
+
+    }
+
+    @Test
+    public void rejectSignOffScenario() {
+
+        when(dcuMinSignOffProcess.waitsAtUserTask(APPROVE_MINISTER_SIGN_OFF))
+                .thenReturn(task -> task.complete(withVariables("valid", true, "MinisterSignOffDecision", "REJECT")));
+
+        when(dcuMinSignOffProcess.waitsAtUserTask(DCU_MIN_MinisterSignOff.VALIDATE_REJECTION_NOTE))
+                .thenReturn(task -> task.complete(withVariables("valid", true, "DIRECTION", "FORWARD")));
+
+
+        Scenario.run(dcuMinSignOffProcess)
+                .startByKey("DCU_MIN_MINISTER_SIGN_OFF")
+                .execute();
+
+        verify(dcuMinSignOffProcess)
+                .hasFinished(VALIDATE_REJECTION_NOTE);
+
+        verify(dcuMinSignOffProcess)
+                .hasFinished(DCU_MIN_MINISTER_SIGN_OFF_END);
+
+        verify(dcuMinSignOffProcess, never()).hasFinished(VALIDATE_NOT_APPLICABLE);
+
+        verify(bpmnService).updateAllocationNote(any(), any(), any(), eq("REJECT"));
```

Comment:
Would be good for the third "any()" here to match against the actual case note text.

Reply:
I did think that at first, but in test it comes through empty. I'll have a look and figure out how to that.

Reply:
"VALIDATE_REJECTION_NOTE" should mock a return value for "CaseNote_MinisterReject"; that value should come through in place of the empty. (Same thing for next unit test below.)

Reply:
added rejection string